test/benchee_test.exs

```diff
@@ -526,6 +526,35 @@ defmodule BencheeTest do
       refute output =~ ~r/ips/i # no runtime statistics displayed
       assert output =~ ~r/memory.+statistics/i
     end
+
+    test "the micro keyword list code from Michal does not break memory measurements #213" do
```

Comment:
So when I was playing around with this example from Michał last night, when I ran just the ones that were messing up, they didn't mess up! I had to run all the scenarios to get these to have those negative memory measurements. So, I think we might need all of them in this test and not just the "problematic" scenarios. Were you seeing something different in your experiments today?

Reply:
Yeah saw something different in my experiments today :) At least on my linux machine it works, it's easy to make it fail - in `Measure.Memory.total_memory/1` just outcomment the `+ old heap size` bit. For me it fails locally as it produces a whole bunch of "N/A" etc. Actually, the large hit doesn't even show up because all 3 are/were botched :)

Ah I even still had the terminal output from trying it out:

```
tobi@comfy:~/github/benchee(master)$ mix run test/support/bench_keyword.ex 
Operating System: Linux"
CPU Information: Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
Number of Available Cores: 4
Available memory: 7.67 GB
Elixir 1.6.4
Erlang 20.3

Benchmark suite executing with the following configuration:
warmup: 0 μs
time: 0 μs
memory time: 1 ms
parallel: 1
inputs: large hit, large miss
Estimated total run time: 6 ms


Benchmarking delete keymember reverse with input large hit...
Benchmarking delete keymember reverse with input large miss...
Benchmarking delete old with input large hit...
Benchmarking delete old with input large miss...
Benchmarking delete reverse with input large hit...
Benchmarking delete reverse with input large miss...

##### With input large hit #####
##### With input large miss #####
Memory usage statistics:

Name                             average  deviation         median         99th %
delete keymember reverse         1.05 KB     ±0.00%        1.05 KB        1.05 KB
WARNING the scenario "delete old" has no memory measurements! This is probably a bug please report it!
https://github.com/PragTob/benchee/issues/new
delete old                           N/A
WARNING the scenario "delete reverse" has no memory measurements! This is probably a bug please report it!
https://github.com/PragTob/benchee/issues/new
delete reverse                       N/A

Comparison: 
delete keymember reverse         1.05 KB
delete old                           N/A
delete reverse                       N/A
```